src/CTA.WebForms2Blazor/MigrationManager.cs

```diff
@@ -21,7 +21,7 @@ public class MigrationManager
 
         private WorkspaceManagerService _webFormsWorkspaceManager;
         private WorkspaceManagerService _blazorWorkspaceManager;
-        private FileInformationFactory _fileFactory;
+        private FileConverterFactory _fileFactory;
```

Comment:
I think Jon made a comment about changing the name to _converterFactory on the last version